src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -243,52 +248,42 @@ import 'emby-ratingbutton';
     }
 
     function showRemoteControl() {
-        import('appRouter').then(({default: appRouter}) => {
+        import('../appRouter').then(({default: appRouter}) => {
```

Comment:
```suggestion
        import('../appRouter').then(({ appRouter }) => {
```
or as usual - import at the top

Reply:
fixed